src/localizations/en.json

```diff
@@ -129,9 +129,11 @@
     "missing_delegable_right": "Missing delegable rights",
     "confirm_delegation_text": "Confirm that you want to give these rights to {{name}}",
     "choose_rights_chip_text": "Choose rights you want to give access to below:",
-    "choose_service_page_top_text": "On this page you can give access to forms and services to {{name}}. We recommend to filter on owner, and to search on name on the service. You can add by picking from the list below.",
+    "delegate_choose_service_page_top_text": "On this page you can give access to forms and services to {{name}}. We recommend to filter on owner, and to search on name on the service. You can add by picking from the list below.",
+    "request_choose_service_page_top_text": "On this page you can request access to services for {{name}}. We recommend you filter on owner or search for the service by name. You can add by picking from the list below.",
```

Comment:
```suggestion
    "request_choose_service_page_top_text": "On this page you can request access to services for {{name}}. We recommend to filter on owner or search for the service by name. You can add by picking from the list below.",
```

Reply:
Hmm.. "We recommend to filter on owner or..." doesn't sound like the best grammar to me. (Although, neither does "we recommend you filter on..." either)
How about: "We recommend that you filter the services based on relevant owners or search for a specific service by name"?

Reply:
Oh, I didn't actually change the text on the line above, but you're right - they should both be changed